lib/workers/global/index.ts

```diff
@@ -69,6 +71,14 @@ function checkEnv(): void {
   }
 }
 
+export async function validatePresets(config: GlobalConfig): Promise<void> {
+  try {
+    await resolveConfigPresets(config);
+  } catch (err) /* istanbul ignore next */ {
+    throw new Error(CONFIG_PRESETS_INVALID);
```

Comment:
trace log error for easier debugging?

Reply:
It already logs in a previous function. Here's an example of what you see:

```
DEBUG: Preset fetch error
       "preset": "local>renovate-tests/preset4",
       "err": {
         "message": "dep not found",
         "stack": "Error: dep not found\n    at fetchJSONFile (/Users/rhys/src/renovatebot/renovate/lib/config/presets/github/index.ts:29:11)\n    at processTicksAndRejections (internal/process/task_queues.js:93:5)\n    at Object.fetchPreset (/Users/rhys/src/renovatebot/renovate/lib/config/presets/util.ts:47:21)\n    at getPreset (/Users/rhys/src/renovatebot/renovate/lib/config/presets/index.ts:170:22)\n    at Object.resolveConfigPresets (/Users/rhys/src/renovatebot/renovate/lib/config/presets/index.ts:245:27)\n    at validatePresets (/Users/rhys/src/renovatebot/renovate/lib/workers/global/index.ts:76:5)\n    at Object.start (/Users/rhys/src/renovatebot/renovate/lib/workers/global/index.ts:90:5)\n    at /Users/rhys/src/renovatebot/renovate/lib/renovate.ts:10:22"
       }
 INFO: Throwing preset error
       "validationError": "Cannot find preset's package (local>renovate-tests/preset4)"
FATAL: Fatal error: config-presets-invalid
       "err": {
         "message": "config-presets-invalid",
         "stack": "Error: config-presets-invalid\n    at validatePresets (/Users/rhys/src/renovatebot/renovate/lib/workers/global/index.ts:78:11)\n    at processTicksAndRejections (internal/process/task_queues.js:93:5)\n    at Object.start (/Users/rhys/src/renovatebot/renovate/lib/workers/global/index.ts:90:5)\n    at /Users/rhys/src/renovatebot/renovate/lib/renovate.ts:10:22"
       }
DEBUG: Renovate exiting
 INFO: Renovate is exiting with a non-zero code due to the following logged errors
       "loggerErrors": [
         {
           "name": "renovate",
           "level": 60,
           "logContext": "TOI1oKmRd",
           "err": {
             "message": "config-presets-invalid",
             "stack": "Error: config-presets-invalid\n    at validatePresets (/Users/rhys/src/renovatebot/renovate/lib/workers/global/index.ts:78:11)\n    at processTicksAndRejections (internal/process/task_queues.js:93:5)\n    at Object.start (/Users/rhys/src/renovatebot/renovate/lib/workers/global/index.ts:90:5)\n    at /Users/rhys/src/renovatebot/renovate/lib/renovate.ts:10:22"
           },
           "msg": "Fatal error: config-presets-invalid"
         }
       ]
error Command failed with exit code 1.
```